specs/sharding/p2p-interface.md

```diff
@@ -77,18 +77,19 @@ Following the same scheme as the [Phase0 gossip topics](../phase0/p2p-interface.
 
 | Name                             | Message Type              |
 |----------------------------------|---------------------------|
-| `shard_blob_{shard}`             | `SignedShardBlob`         |
+| `shard_blob_{shard}_{slot}`      | `SignedShardBlob`         |
```

Comment:
@protolambda does it make sense to make the `shard_blob_{subnet_id}` matching the `beacon_attestation_{subnet_id}` (with respect to `(slot, shard)`) to optimize peers discovery?
I.e. when a validator subscribes to `shard_blob_N` and `beacon_attestation_N` for the corresponding `(slot, shard)` it would need the same subset of peers to be discovered and connected

Reply:
I'd use the same subnet_id calculation yes, so that discovery is easy (assuming most nodes are validators, and on the attestation subnets that match the blob subnets). Later on we probably still should have an explicit field (e.g. `blobnets`) for it in the ENR (like `attnets`).